docs/react-native-devtools.md

```diff
@@ -0,0 +1,99 @@
+---
+id: react-native-devtools
+title: React Native DevTools
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+React Native DevTools is our new debugging experience featuring an end-to-end rewrite of our debugger stack. It aims to be more deeply integrated and fundamentally more reliable than previous debugging methods in React Native.
```

Comment:
```suggestion
React Native DevTools is the tool you can use to debug React Native apps.
It aims to be more deeply integrated and fundamentally more reliable than previous debugging methods in React Native.
```

I will remove all the sentences with a temporal connotation as they will not age well. Years from now, the page will still read as "our new debugging experience" and it will talk about the complete rewrite, even if that happened years ago.

Reply:
https://github.com/facebook/react-native-website/pull/4275#discussion_r1804709643

Reply:
I don't get the reply. I really think we need to remove all the temporal references or it will be our "new debugging experience" also in 0.80.